src/s1reader/s1_reader.py

```diff
@@ -349,28 +394,47 @@ def burst_from_xml(annotation_path: str, orbit_path: str, tiff_path: str,
     # burst sensing starts in prototyping test data
     burst_interval = 2.758277
 
-    #parse manifest.safe to retrieve IPF version
+    # parse manifest.safe to retrieve IPF version
     manifest_path = os.path.dirname(annotation_path).replace('annotation','') + 'manifest.safe'
     with open_method(manifest_path, 'r') as f_manifest:
         tree_manfest = ET.parse(f_manifest)
         ipf_version = get_ipf_version(tree_manfest)
 
-    #Load the Product annotation - for EAP calibration
-    #with open_method(annotation_path, 'r') as f_lads:
-    #    tree_lads = ET.parse(f_lads)
-    #    product_annotation = s1_annotation.ProductAnnotation.from_et(tree_lads)
+    # Load the Product annotation - for EAP calibration
+    with open_method(annotation_path, 'r') as f_lads:
+        tree_lads = ET.parse(f_lads)
+        product_annotation = ProductAnnotation.from_et(tree_lads)
 
-    #load the Calibraton annotation
-    calibration_annotation_path = annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
+    # load the Calibraton annotation
+    calibration_annotation_path =\
+        annotation_path.replace('annotation/', 'annotation/calibration/calibration-')
```

Comment:
You should avoid using forward slashes to separate directories. This would not work in a windows machine. Ideally, you'd do the operations from `os.path`, for example, instead of `'annotation/calibration/calibration-'`, you'd have something like `os.path.join('annotation', ',calibration', 'calibration-').

Reply:
In the context of OPERA, since workflows dependent on `s1-reader` all use `isce3`, which does not support Windows, I don't think it a good use of time and effort to accommodate Windows natively in `s1-reader`.

If someone really wants to run this on Windows, they can install WSL2 and install Conda and `s1-reader` there.

Reply:
@gshiroma if you think native Windows support is vital, perhaps open an issue or start a discussion in this repo? This way we don't derail discussion in this PR.

Reply:
Hi @LiangJYu , yes, we are not supporting Windows. I just think it's a good practice to avoid using `/` as a directory separator. That way it would be easier if someday we decide to make certain pieces of code/modules working in other machines. Or it would be easier for other developers to do so, even if it's not our goal.

Reply:
@gshiroma `calibration_annotation_path` can be path inside .zip file when S1 data is provided as a .zip file. The path will be in the file system when the directory to (unzipped) SAFE directory is provided.

I've found the article below, and wanted to check how the path looks like with Windows Python, but I do not have one handy right now.

https://stackoverflow.com/questions/8176953/python-zipfile-path-separators

If that is true, we should differentiate the separator (i.e. forward- or back slash) depending on the SAFE file form (.zip or directory), as well as the platform (UNIX-like, or Windows).

Reply:
From the reference you pointed out, I believe that it expects `os.sep` (which is the system separator). So, you wouldn't need to differentiate them. You can just use `os.sep` and `os.path.join`.

Reply:
Managed to get a Windows machine and do some test. It turned out that we use forward slash when referring to the file in a .zip file. in Windows. Therefore `os.path.join` is not going to work on Windows machine when attempting to load bursts from S1 .zip file.

I totally agree that `os.path.join` is better practice in general. But our s1-reader has an option to load directly from .zip file. in that case, we need to use `/` regardless of the platform as in the table below:

| | Unix-like |  Windows|
|----|--------|-----------|
| S1 data as .zip| / | / |
|S1 data as directory| / | \ |

This implies that it takes more effort than replacing the line with `os.path.join` to make this line compatible with other platform (i.e. Windows). Of course using the join function would work okay with our current target platform (linux), but not sure if we this is absolutely necessary.